app/src/main/java/com/sunasterisk/englishflashcard/ui/exam/ExamFragment.kt

```diff
@@ -0,0 +1,8 @@
+package com.sunasterisk.englishflashcard.ui.exam
+
+import com.sunasterisk.englishflashcard.R
+import com.sunasterisk.englishflashcard.ui.BaseFragment
+
+class ExamFragment : BaseFragment() {
+    override val layoutResource = R.layout.fragment_exam
```

Comment:
viết như này thì nó chỉ kế thừa hàm get thôi k phải sinh thêm thuộc tính k cần thiết 
```suggestion
    override val layoutResource get() = R.layout.fragment_exam
```